awx/ui/client/src/projects/add/projects-add.controller.js

```diff
@@ -170,6 +170,14 @@ export default ['$scope', '$location', '$stateParams', 'GenerateForm',
                             $scope.lookupType = 'scm_credential';
                             $scope.scmBranchLabel = i18n._('SCM Branch/Tag/Revision');
                             break;
+                        case 'archive':
+                            $scope.credentialLabel = "SCM " + i18n._("Credential");
+                            $scope.urlPopover = '<p>' + i18n._('Example URLs for Remote Archive SCM include:') + '</p>' +
+                                '<ul class=\"no-bullets\"><li>https://github.com/philipsd6/test-playbook/archive/v0.0.1.tar.gz</li>' +
+                                '<li>http://github.com/philipsd6/test-playbook/archive/v0.0.2.zip</li></ul>';
```

Comment:
I get the idea here, but we'd need to replace this with either a generic example URL that doesn't actually exist, or an upstream URL/reference.

Reply:
OK, how about I just update these to `s/philipsd6/username/` to match the style in the Mercurial sample.

Reply:
I updated these URLS to be:
* https://github.com/username/project/archive/v0.0.1.tar.gz
* http://github.com/username/project/archive/v0.0.2.zip

Reply:
That works for me.